scripts/fix-ts-types.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * This script removes unnecessary and broken types from "types.d.ts" file.
+ * Script should be run on repository's root path.
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const tsFile = path.resolve(__dirname, '../types.d.ts');
+
+const data = fs.readFileSync(tsFile, 'utf-8');
+const namespaceRegex = /(declare namespace)[ A-Za-z]*{((.|\n)+?)(?!(};))}/g;
```

Comment:
sorry but I'm regex looser, can you add a comment with example of what values being replaced with empty string

Reply:
Added example in file description :)